build.gradle.kts

```diff
@@ -1,5 +1,12 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
+    id("io.cloudflight.autoconfigure-gradle") version "1.0.0"
```

Comment:
i am not quite sold on using these plugins, what exactly are the doing and why do we need them?